common/common.go

```diff
@@ -102,6 +102,13 @@ func PodManifestPath(root string) string {
 	return filepath.Join(root, "pod")
 }
 
+// PodManifestLockPath returns the path in root to the Pod Manifest lock file.
+// This must be different from the PodManifestPath since mutations on the pod manifest file
```

Comment:
Ok, I guessed so but the comment may be a bit more explicit as casual readers won't have the whole context in mind.